staker/staker.go

```diff
@@ -280,8 +289,42 @@ func (s *Staker) Initialize(ctx context.Context) error {
 	return nil
 }
 
+func (s *Staker) latestNodeDetailsForUpdate(ctx context.Context, description string, node uint64) (arbutil.MessageIndex, *validator.GoGlobalState, error) {
+	stakedInfo, err := s.rollup.LookupNode(ctx, node)
+	if err != nil {
+		return 0, nil, fmt.Errorf("couldn't look up latest %v assertion %v: %w", description, node, err)
+	}
+
+	globalState := stakedInfo.AfterState().GlobalState
+	caughtUp, count, err := GlobalStateToMsgCount(s.inboxTracker, s.txStreamer, globalState)
+	if err != nil {
+		if errors.Is(err, ErrGlobalStateNotInChain) && s.fatalErr != nil {
+			fatal := fmt.Errorf("latest %v assertion %v not in chain: %w", description, node, err)
+			s.fatalErr <- fatal
+		}
+		return 0, nil, fmt.Errorf("latest %v assertion %v: %w", description, node, err)
+	}
+
+	if !caughtUp {
+		log.Info(fmt.Sprintf("latest %v assertion not yet in our node", description), "assertion", node, "state", globalState)
```

Comment:
not a fan of sprintf for first arg of log.Info. Hard to grep. you could print "assertionType" as another field.

Reply:
I've fixed this by logging the staker address instead